.github/workflows/image-push.yml

```diff
@@ -0,0 +1,29 @@
+name: Build and Push Images
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+permissions:
+  contents: read
+  id-token: write
+jobs:
+  price-service-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::192824654885:role/github-actions-ecr
+          aws-region: eu-west-2
+      - uses: aws-actions/amazon-ecr-login@v1
+        id: ecr_login
+      - run: |
+          DOCKER_BUILDKIT=1 docker build -f Dockerfile.wasm -o type=local,dest=. .
+          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG -f third_party/pyth/price-service/Dockerfile.price_service .
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+        env:
+          ECR_REGISTRY: ${{ steps.ecr_login.outputs.registry }}
+          ECR_REPOSITORY: price-service
```

Comment:
Can you set this to `pyth-price` so we can reuse the repository that has already been created? We can then later revisit the names for the three p2w repositories.

Reply:
Ah you are right. i thought it was price-service in first place :D fixed.